python/ray/tests/test_multi_node.py

```diff
@@ -11,75 +11,6 @@
     wait_for_children_of_pid_to_exit, kill_process_by_name, Semaphore)
 
 
-def test_error_isolation(call_ray_start):
```

Comment:
We shouldn't remove this test. We should test if error msgs are still streamed to different drivers correctly.